lib/chain/ChainClient.ts

```diff
@@ -32,13 +37,34 @@ class ChainClient extends EventEmitter implements ChainClientInterface, ChainCli
   }
 
   public connect = async () => {
-    await this.rpcClient.connect();
-
-    this.bindWs();
+    if (this.isDisconnected) {
+      await this.rpcClient.connect();
+      try {
+        const info = await this.getInfo();
+        if (info.blocks) {
```

Comment:
```suggestion
        if (info.version) {
```

Use the version instead of the blocks so that there is no error when connecting to a BTCD/LTCD without any blocks like the instances we start for the tests.